pkg/repo/repo.go

```diff
@@ -32,4 +32,6 @@ type MessagesRepo interface {
 type Repo interface {
 	ServicesRepo
 	MessagesRepo
+
+	Close() error
```

Comment:
**Quick question:**
Are there any advantages of using `io.Closer,` instead of `Close() func` ?

Reply:
As you know, @VladislavsPerkanuks , golang has implicit interfaces. In general, both ways are fine. However, if an implementation conforms to an interface, it does not meet they have the same intent. As a rule of thumb, use an interface if it matches the intent.